src/python/TaskWorker/Worker.py

```diff
@@ -97,6 +97,14 @@ def processWorkerLoop(inputs, results, resthost, resturi, procnum, logger):
                     logger.exception('Traceback follows:')
         t1 = time.time()
         logger.debug("%s: ...work on %s completed in %d seconds: %s", procName, task['tm_taskname'], t1-t0, outputs)
+
+        try:
+            out, _, _ = executeCommand("ps u -p %s | awk '{sum=sum+$6}; END {print sum/1024}'" % os.getpid())
```

Comment:
No, I don't think it's anywhere in CRAB, that script looks like something to be used standalone. Anyway, I'll merge this for now so that I can deploy it on pre-prod, I'll make a PR later if this isn't good enough for us.